test/fgpu/test_engine.py

```diff
@@ -913,9 +915,60 @@ async def test_missing_heaps(
                 # the time difference is a multiple of the mixer wavelength.
                 np.testing.assert_equal(x, y)
 
+        # Information needed to check that the dig-rms-dbfs sensor is correctly
+        # going into FAILURE when there are missing samples.
+        time_converter = TimeConverter(SYNC_TIME, ADC_SAMPLE_RATE)
+        unix_timestamps = [time_converter.adc_to_unix(t) for t in timestamps]
+        # The sensor is updated once per out_item.
+        timestamps_per_output_chunk = engine_server.chunk_jones // output.channels
```

Comment:
"timestamps" is not a helpful name here - there are many different objects (ADC samples, spectra, heaps) that have associated timestamps and there are different numbers of them per output chunk. If I'm reading the code right, this is `spectra_per_output_chunk`?

Reply:
You're correct, that's probably a better way to refer to it.